core/src/main/java/org/apache/sdap/mudrod/ontology/pre/AggregateTriples.java

```diff
@@ -62,30 +62,28 @@ public Object execute() {
       e2.printStackTrace();
     }
 
-    FileWriter fw;
-    try {
-      fw = new FileWriter(file.getAbsoluteFile());
+    try(FileWriter fw = new FileWriter(file.getAbsoluteFile())){
       bw = new BufferedWriter(fw);
-    } catch (IOException e) {
-      e.printStackTrace();
-    }
-
-    File[] files = new File(this.props.getProperty(MudrodConstants.ONTOLOGY_INPUT_PATH)).listFiles();
-    for (File file_in : files) {
-      String ext = FilenameUtils.getExtension(file_in.getAbsolutePath());
-      if ("owl".equals(ext)) {
-        try {
-          loadxml(file_in.getAbsolutePath());
-          getAllClass();
-        } catch (JDOMException e1) {
-          e1.printStackTrace();
-        } catch (IOException e1) {
-          e1.printStackTrace();
+      File[] files = new File(this.props.getProperty(MudrodConstants.ONTOLOGY_INPUT_PATH)).listFiles();
+      for (File file_in : files) {
+        String ext = FilenameUtils.getExtension(file_in.getAbsolutePath());
+        if ("owl".equals(ext)) {
+          try {
+            loadxml(file_in.getAbsolutePath());
+            getAllClass();
+          } catch (JDOMException e1) {
+            e1.printStackTrace();
+          } catch (IOException e1) {
+            e1.printStackTrace();
+          }
+  
         }
-
       }
+    } catch (IOException e) {
+      // TODO Auto-generated catch block
```

Comment:
Never leave comment stubs present in production level code. Either LOG or rethrow this exception.